index.html

```diff
@@ -2825,125 +2827,265 @@ <h3>Working with personal names</h3>
 </aside>
 
 
+<section id="loc_names_field_length">
+<h4>Field length &amp; composition</h4>
+
 	<div class="req" id="loc_names_given_family">
-	<p class="advisement">Check whether you really need to store or access given name and family name separately.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<p class="advisement">Check whether you really need to store or access given name(s) and family name(s) separately.</p>
```

Comment:
I actually added the (s) here – they weren't in the original.  Indeed, it's to emphasise that there can be more than one of each.  However, saying "access given names" loses that message because the plurality can be applied to the idea that there are  multiple person's names. So i'm not keen on changing that.

Reply:
Sounds good.